packages/expo-cli/scripts/introspect.ts

```diff
@@ -612,87 +592,8 @@ if (['markdown', 'md'].includes(process.argv[2])) {
       }),
     };
 
-    // Replace all quotes around generators
-    const parsed = replaceAll(
-      JSON.stringify(figSpec, null, 2),
-      /"(_gen\[`.*`\])"/g,
-      (_: string, inner: string) => {
-        return inner;
-      }
-    );
-
-    const contents = `
-    // Copyright (c) 2021-present 650 Industries, Inc. (aka Expo)
-    // Generated by running \`yarn introspect fig\` in expo-cli/packages/expo-cli
-    // expo-cli@${require('../package.json').version}
-  
-    const _gen: Record<string, Fig.Generator> = {
-      "npm": {
-        script(context) {
-          if (context[context.length - 1] === "") return "";
-          const searchTerm = context[context.length - 1];
-          return \`curl -s -H "Accept: application/json" "https://api.npms.io/v2/search?q=$\{searchTerm}&size=20"\`;
-        },
-        postProcess(script: string) {
-          try {
-            const results: {
-              package: {
-                name: string;
-                description: string;
-              };
-              searchScore: number;
-            }[] = JSON.parse(script).results;
-            return results.map((item) => ({
-              name: item.package.name,
-              description: item.package.description,
-            })) as Fig.Suggestion[];
-          } catch (e) {
-            return [];
-          }
-        },
-      },
-      "xcode-configuration": {
-        script: "xcodebuild -project ios/*.xcodeproj -list -json",
-        postProcess: (script: string) => JSON.parse(script).project.configurations.map((name) => ({ name })),
-      },
-      "xcode-scheme": {
-        script: "xcodebuild -project ios/*.xcodeproj -list -json",
-        postProcess: (script: string) => JSON.parse(script).project.schemes.map((name) => ({ name })),
-      },
-      "xcode-devices": {
-        script: "xcrun xctrace list devices",
-        postProcess: (script: string) =>
```

Comment:
For diff-versioned specs which is a new format we extract generators to a separator file which, in this case is shared between the expo@4 and expo@5 specs